ci-operator/config/openshift-assisted/assisted-installer-ui/openshift-assisted-assisted-installer-ui-master.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  node18:
+    name: nodejs-18-minimal
+    namespace: edge-infrastructure
+    tag: latest
+  openshift_release_golang-1.21:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+  openshift_ubi-micro_9:
+    name: ubi-micro
+    namespace: ocp
+    tag: "9"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+images:
+- dockerfile_path: apps/assisted-disconnected-ui/Containerfile
+  to: assisted-disconnected-ui
```

Comment:
In future this name will be relevant, as it will refer the promoted  image. I'd expected something like `agent-install-ui` to be in line with the other OCP agent related payload

Reply:
Our repo is a monorepo that contains multiple apps. The app that we are building here is `assisted-disconnected-ui` and that is also how we would promote it. Does that make sense ?

Reply:
The promotion step is used as a (preliminary) step for generating an image of the OCP release payload, so the focus here is on the specific container file used for generating such image (it's not strictly relevant if the underlying repo is a monorepo or not). The image name instead will make sense in future for the OCP content.

For example, assisted-service OCP image is promoted as [agent-installer-api-server](https://github.com/openshift/release/blob/d459b408b43b761dea4d2a7eaa2c70909d49d96f/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml#L17-L25). Also, to make more evident that such particular containerfile is dedicated for building an OCP image, we adopted the convention of adding an `.ocp` suffix

Reply:
Thanks for explanation. I will adjust.